python/GafferImageUI/ResampleUI.py

```diff
@@ -81,14 +81,16 @@
 
 		) ),
 
-		"filterWidth" : [
+		"filterScale" : [
 
 			"description",
 			"""
-			An override for the width of the filter used. This is
+			An override for the scale of the filter used. This is
```

Comment:
OK, cool - that's good news. I thought maybe I'd got totally the wrong end of the stick about what you were doing. I think I'd be content just to see this described as a multiplier on the filter's natural width, and consider the rest to be internal implementation details. The only client of this plug so far is the Blur node, and that description is just about sufficient to inform the approach taken there.